packages/lsd-react/src/components/Calendar/Calendar.classes.ts

```diff
@@ -0,0 +1,22 @@
+export const calendarClasses = {
+  root: `lsd-calendar`,
+  container: 'lsd-calendar-container',
+
+  open: 'lsd-calendar--open',
+  disabled: 'lsd-calendar--disabled',
+
+  header: 'lsd-calendar-header',
+  grid: 'lsd-calendar-body',
+  weekDay: 'lsd-calendar__weekDay',
+  button: 'lsd-calendar-button',
+  row: 'lsd-calendar__row',
+  changeYear: 'lsd-calendar__change-year',
+  changeYearButton: 'lsd-calendar__change-year__button',
+
+  year: 'lsd-calendar-year',
+  month: 'lsd-calendar-month',
+  day: 'lsd-calendar-day',
+  daySelected: 'lsd-calendar-day--selected',
+  dayDisabled: 'lsd-calendar-day--disabled',
+  today: 'lsd-calendar-today',
```

Comment:
Thanks. I totally agree with `lsd-calendar-day__today` but I didn't fully understand why `dayIsToday` is needed and where it could be used. Can you please explain a bit more?

Reply:
New class dayIsToday added. If this was not what you were thinking about, let me know.